packages/ember-engines/package.json

```diff
@@ -125,8 +125,7 @@
     "lodash": "^4.17.11"
   },
   "peerDependencies": {
-    "ember-engines-router-service": ">=0.5.0",
-    "ember-source": "^3.28 || >= 4.0.0"
```

Comment:
complete removal of ember-source as a peer? seems... scary 😂 maybe just expand the range?